server/game/gamesteps/plot/selectplotprompt.js

```diff
@@ -50,9 +51,30 @@ class SelectPlotPrompt extends AllPlayerPrompt {
         }
 
         player.selectedPlot = plot;
+        player.clearSelectableCards();
 
         this.game.addMessage('{0} has selected a plot', player);
     }
+
+    highlightSelectableCards(player) {
+        let selectableCards = this.game.allCards.filter(card => card.getType() === 'plot' &&
+                                                                card.location === 'plot deck' &&
+                                                                card.controller === player &&
+                                                                (player.cannotSelectSchemes && !card.hasTrait('scheme')));
```

Comment:
This check is causing all plots to be displayed as 'unselectable' when playing with a non-Rains agenda. It still lets the player to select the plot, so it's strictly a UI bug.

Reply:
Oops.